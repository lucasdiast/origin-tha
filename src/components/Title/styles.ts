import styled from 'styled-components';

export const Container = styled.div`
  text-align: center;
  font-size: 18px;
  margin-bottom: 24px;
  letter-spacing: 2px;
  color: ${(props) => props.theme.colors.originBlue};
`;
