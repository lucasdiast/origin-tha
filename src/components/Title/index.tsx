import React, { ReactElement, ReactChild } from 'react';
import { Container } from './styles';

interface AuxProps {
  children: ReactChild;
}

const Title = ({ children }: AuxProps): ReactElement => (
  <Container>{children}</Container>
);

export default Title;
