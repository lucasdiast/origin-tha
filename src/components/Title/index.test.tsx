import { shallow } from 'enzyme';
import Title from './index';

describe('Title', () => {
  it('renders correctly', () => {
    const component = shallow(<Title>Some text</Title>);

    expect(component).toMatchSnapshot();
  });
});
