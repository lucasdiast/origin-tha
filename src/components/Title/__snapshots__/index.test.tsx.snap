// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `ShallowWrapper {}`;
